add chapter to describe how to acquire permission (#40116)
Azure/azure-sdk-for-net: sdk/core/Azure.Core.TestFramework/README.md

On completion of the push, a newly created tag will be stamped into the `assets.json` file. This new tag must be committed and pushed to your package directory along with any other changes.

> ☑️ Permission is required for updating the asserts repository. If you failed in test-proxy push, please [join a partner write team](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/785/Externalizing-Recordings-(Asset-Sync)?anchor=permissions-to-%60azure/azure-sdk-assets%60).

Reviewer: ```suggestion
☑️ Permission is required for updating the assets repository. If you failed when executing `test-proxy push` command, please [join a partner write team](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/785/Externalizing-Recordings-(Asset-Sync)?anchor=permissions-to-%60azure/azure-sdk-assets%60).
```

> ☑️ Permission is required for updating the assets repository. If you failed when executing `test-proxy push` command, please [join a partner write team](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/785/Externalizing-Recordings-(Asset-Sync)?anchor=permissions-to-%60azure/azure-sdk-assets%60).

Reviewer 2: This is great for internal contributors with access to the linked documentation, however, that is not a public link and non-Microsoft contributors cannot access it.   

We really need an approach for this that allows community contributors to update test assets, but I don't believe that one exists at this time.  Assuming that hasn't changed, can we please add a note that this documentation is "Microsoft Internal" and community contributors will need to work with the reviewers for their PR to get assets merged in?

@JoshLove-msft, @scbedd:  Do we have any update here for community contributor support?

Author: Thanks @jsquire I add the notes now. And we may update later if there is a wiki for community contributor. thanks

> ☑️ Permission is required for updating the assets repository. If you failed when executing `test-proxy push` command, please [join a partner write team](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/785/Externalizing-Recordings-(Asset-Sync)?anchor=permissions-to-%60azure/azure-sdk-assets%60).

Reviewer 2: What's the reason for using this icon?  I don't see this convention used elsewhere in the doc, and this doesn't really highlight and call attention to this, in my opinion.  It seems more appropriate for defining a checklist.

I'd rather see us annotate with **NOTE:** and/or some kind of bit exclamation point.

Author: Thanks @jsquire I updated to **NOTE**.
